chaos_kitten/brain/postman_parser.py

```
@@ -0,0 +1,310 @@
+                language = request['body'].get('options', {}).get('raw', {}).get('language', 'json')
+                raw_data = self._resolve_variables(request['body'].get('raw', ''))
+                
+                # Default to text/plain, try to infer json
+                content_type = "text/plain"
+                schema = {"type": "string"}
+                
+                if language == "json":
+                    content_type = "application/json"
+                    # Try to parse example as schema if it's valid json
+                    try:
+                        json_data = json.loads(raw_data)
```

Ruff (0.15.2)

[error] Local variable `json_data` is assigned to but never used

Remove assignment to unused variable `json_data`

(F841)